webdriver/navigation.py

```diff
@@ -1,109 +1,96 @@
 import json
 import pytest
 import types
-import urllib
-
-import webdriver
-
-
-def inline(doc):
-    return "data:text/html;charset=utf-8,%s" % urllib.quote(doc)
 
+from util.inline import inline
+from util import wd_assert
 
 alert_doc = inline("<script>window.alert()</script>")
 frame_doc = inline("<p>frame")
 one_frame_doc = inline("<iframe src='%s'></iframe>" % frame_doc)
 two_frames_doc = inline("<iframe src='%s'></iframe>" % one_frame_doc)
 
 
-@pytest.fixture
-def new_window(session):
```

Comment:
This is a helpful fixture! This patch re-locates it to a separate module so we can re-use it from other tests. Those aren't included here, but I have some complete already that I plan to submit as a follow-up.